packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/layout_explorer_widget.dart

```diff
@@ -80,6 +80,8 @@ abstract class LayoutExplorerWidgetState<W extends LayoutExplorerWidget,
       final newSelection = await fetchLayoutProperties();
```

Comment:
It looks like `fetchLayoutProperties` calls `WidgetInspectorServiceExtensions.getLayoutExplorerNode` is called through objectGroups.
https://github.com/flutter/devtools/blob/54111ea25e1067a1735bac07b888f90d85a7df37/packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/layout_explorer_widget.dart#L107

Reply:
Here is the relevant code on the Flutter side. It is using the Element.renderObject getter.
          final RenderObject? renderObject = value is Element ? value.renderObject : null;
https://github.com/flutter/flutter/blob/c8265f81cf591587f6056a9f9def6150008d9b6e/packages/flutter/lib/src/widgets/widget_inspector.dart#L2070C64-L2070C64
and then adding some properties if the render object has been laid out.

https://github.com/flutter/flutter/blob/c8265f81cf591587f6056a9f9def6150008d9b6e/packages/flutter/lib/src/widgets/widget_inspector.dart#L2102
We're just

Reply:
Makes sense, findRenderObject is implemented in terms of the renderObject getter (with some additional checks). so this would behave as expected then. If this gives you null we shouldn't show the layout explorer.